src/services/db/review.ts

```diff
@@ -129,3 +129,16 @@ export const createComment = async (
     { body: stringifiedMessage }
   )
 }
+
+export const getBlob = async (
+  repo: string,
+  path: string,
+  ref: string
+): Promise<string> =>
+  axiosInstance
+    .get<string>(`${repo}/contents/${path}?ref=${ref}`, {
+      headers: {
+        Accept: "application/vnd.github.VERSION.raw",
```

Comment:
Should we remove the `VERSION` part of this header? I was not able to find it [in the docs](https://docs.github.com/en/rest/overview/media-types?apiVersion=2022-11-28).

Reply:
huh that's weird - you're right, idk where i saw this in the docs but since it's not in the latest one, i'll remove it. thanks for checking!